### PR TITLE
ttl: add system table for TTL status

### LINKED ISSUE #39272: Add system table `mysql.tidb_ttl_table_status`

### PR DESCRIPTION
**Pull request by YangKeao**
### What problem does this PR solve?

Issue Number: close #39272

Initialize the system table for TTL.

### What is changed and how it works?

This PR does the following things:

1. Create a status table for TTL as described in RFC. https://github.com/pingcap/tidb/pull/39264/files#diff-3199b62f6b171f846c9c0c1670428817d339e0e2ee2b2a35822df8f532c51e58R97

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

### Release note

```release-note
Add a system table to record ttl job status.
```

### CODE REVIEW

File: session/bootstrap.go [hunk 1]
```diff
@@ -2181,6 +2199,8 @@ func doDDLWorks(s Session) {
 	mustExecute(s, CreatePlanReplayerTaskTable)
 	// Create stats_meta_table_locked table
 	mustExecute(s, CreateStatsTableLocked)
+	// Create tidb_ttl_table_status table
+	mustExecute(s, CreateTTLTableStatus)
```
Comment (lcwangchao) [inline session/bootstrap.go:2203]: I think we should also add some codes to create a table when upgrading from an old version.

Comment (YangKeao) [reply session/bootstrap.go:2212]: Done

**Review by lcwangchao — APPROVED** [review]

**Comment by YangKeao** [comment]
@lcwangchao Should we unhold this PR?

@bb7133 PTAL

**Review by bb7133 — APPROVED** [review]
LGTM

**Comment by YangKeao** [comment]
Oops. Forget to unhold the PR :facepalm:

**Review by hawkingrei — APPROVED** [review]

**Pull request merged**